### PR TITLE
fix(common): ldml fix TS version 🙀

### PR DESCRIPTION
**Pull request by srl295**
Fixes: #8420 

- `npm --workspaces=true install --save-dev typescript@^4.9.5`
- manually revert some package.json files where it didn't make sense
- `npm i` for good measure (cleanup said reversion)

@keymanapp-test-bot skip

### CODE REVIEW

**Comment by jahorton** [comment]
Apologies for the partially-duplicated work, but I have a version of this up now as #8429; it addresses a few other critical aspects.  I may have missed a few details here, though.  I'll try to fold them in if that's preferred.

**Comment by srl295** [comment]
> Apologies for the partially-duplicated work, but I have a version of this up now as #8429; it addresses a few other critical aspects. I may have missed a few details here, though. I'll try to fold them in if that's preferred.

up to you ,feel free to close if needed

**Review by mcdurdin — COMMENTED** [review]
@jahorton is this going to conflict with #8429?

File: developer/src/kmc-model/package.json [hunk 1]
```diff
@@ -30,7 +30,6 @@
   "dependencies": {
     "@keymanapp/keyman-version": "*",
     "@keymanapp/models-types": "*",
-    "typescript": "^4.9.5",
     "xml2js": "^0.4.19"
   },
```
Comment (mcdurdin) [inline developer/src/kmc-model/package.json:34]: Typescript needs to be a `dependency` not a `devDependency` because we use typescript to compile models.

**Review by mcdurdin — CHANGES_REQUESTED** [review]

**Comment by jahorton** [comment]
> @jahorton is this going to conflict with #8429?

Hmm.  I don't _think_ it will; the worst difference is that it reorders a few entries, I think.  Easy eyeball review.  Only reason I didn't close it is because the base branch isn't `master`, so I couldn't directly this.#8427 

I believe it's safe to close this PR at this point.

**Comment by srl295** [comment]
@jahorton  "directly … this" ?

**Pull request closed without merging**